src/test/java/racingcar/domain/RacingGameTest.java

```diff
@@ -0,0 +1,24 @@
+package racingcar.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGameTest {
+
+    private Util util = new Util();
+
+    private List<Car> carInfo = new ArrayList<>();
+
+    @DisplayName("자동차 이름이 5자를 초과하는 경우 예외가 발생한다.")
+    @ParameterizedTest
+    @ValueSource(strings = {"jeonggggg,hyeon,jin"})
+    public void splitString(String input) {
+        assertThrows(IllegalArgumentException.class,()->util.invalidedName(input));
```

Comment:
src 디렉토리 혹은 test 디렉토리 클릭하여 자동 정렬을 전체적으로 진행할 수도 있으니, 코드 정렬은 습관적으로 진행해주세요 🙏 
```suggestion
        assertThrows(IllegalArgumentException.class, () -> util.invalidedName(input));
```